bibcheck/plugins/convert_grid_to_ror.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+##
+## This file is part of Invenio.
+## Copyright (C) 2013, 2019 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+"""Bibcheck plugin to convert institution IDs from GRID to ROR."""
+
+import json
+
+GRID_TO_ROR_PATH = '/opt/cds-invenio/var/data/grid_to_ror.json'
```

Comment:
Is this a good place to put this file? I guess the invenio way is to use a knowledge base, but that seemed overly complex to me.

Reply:
reading a 4MB file with 96000 entries every time an Institutions record was modified. The Institutions collection doesn't change that frequently, so most of the time loading this data is for one or two lookups after the initial run. So it certainly seems wasteful compared to a key/value kb, i.e. a simple mysql table and a quick lookup. 

On the other hand the local lookup is probably faster and doesn't put load from a non-interactive task on the mysql db for the initial run.

`'/opt/cds-invenio/var/data/grid_to_ror.json'` is a reasonable choice. However this PR doesn't contain that file and there is no source for it specified and the plugin doesn't handle read failure:
`IOError: [Errno 2] No such file or directory: '/opt/cds-invenio/var/data/grid_to_rorr.json'`

So this can't be merged.